pybotx_widgets/base.py

```diff
@@ -66,24 +64,14 @@ async def send_or_update_message(
     ) -> None:
         """Send new message or update exist."""
 
-        current_message_id = self.message.data.get("message_id")
-        new_message_id = new_message_id or uuid4()
-
-        for bubbles in widget_msg.markup.bubbles:
-            for bubble in bubbles:
-                bubble.data["message_id"] = current_message_id or new_message_id
-
-        if current_message_id:
-            payload = UpdatePayload(text=widget_msg.text, file=widget_msg.file)
-            payload.set_markup(markup=widget_msg.markup)
-            await self.bot.update_message(
-                SendingCredentials(
-                    sync_id=current_message_id,
-                    bot_id=self.message.bot_id,
-                    host=self.message.host,
-                ),
-                update=payload,
-            )
-        else:
-            widget_msg.credentials.message_id = new_message_id
-            await self.bot.send(widget_msg)
+        message_id = self.message.data.get("message_id")
+        is_message_id_exists = bool(message_id)
+
+        message_id = new_message_id or message_id or uuid4()
+
+        widget_msg.credentials.message_id = message_id
```

Comment:
`widget_msg.credentials.message_id `: Optional[UUID].  
`message_id`: Union[str, UUID]
It will pass if `message_id `: str?

Reply:
yep, it's work fine